src/molecule/model/schema_v3.py

```diff
@@ -367,9 +367,9 @@ def _validate_molecule_env_var(self, molecule_env_var, field, value):
                 self._error(field, msg)
 
 
-def pre_validate(stream, env: MutableMapping, keep_string: str):
+def pre_validate(stream, env: MutableMapping, keep_string: str, include_path=""):
```

Comment:
Missing type declaration like you included elsewhere?

Reply:
Fixed